commons/src/main/java/info/metadude/android/eventfahrplan/commons/Clock.kt

```diff
@@ -0,0 +1,42 @@
+package info.metadude.android.eventfahrplan.commons
+
+import org.threeten.bp.LocalDateTime
+import org.threeten.bp.ZoneOffset
+import org.threeten.bp.ZonedDateTime
+
+interface Clock {
+
+    val year: Int
+
+    val month: Int
+
+    val monthDay: Int
+
+    val hour: Int
+
+    val minute: Int
+
+    val minuteOfDay: Int
+
+    fun setToNow()
+
+    fun setToMilliseconds(milliseconds: Long)
+
+    fun toMilliseconds(): Long
+
+    fun toUTCDateTime(): LocalDateTime
```

Comment:
# Naming conventions
Please rename `toUTCDateTime()` to `toUtcDateTime().

Reply:
I think changing a wide spread naming convention (UTC with 100+ results vs Utc with 25 results) should not be done in this PR.

Reply:
I agree. Thanks.